src/NServiceBus.AcceptanceTests/Performance/MessageDurability/When_disabling_durable_messages_on_durable_transport.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace NServiceBus.AcceptanceTests.Performance.MessageDurability
+{
+    using System;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using FakeTransport;
+    using NUnit.Framework;
+
+    public class When_disabling_durable_messages_on_durable_transport : NServiceBusAcceptanceTest
```

Comment:
yes that would fit better and I'm pretty sure I had it this way, but the test ends up having a too long name (since the generated test queues are limited to somewhat 77 chars.